src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -623,11 +621,11 @@ def calculate_average_aspect(
 
     # the flow_lengths array is the functional equivalent
     # of calculating |sin(alpha)| + |cos(alpha)|.
-    cdef float* flow_lengths = [
-        1.0, <float>SQRT2,
-        1.0, <float>SQRT2,
-        1.0, <float>SQRT2,
-        1.0, <float>SQRT2
+    cdef float * flow_lengths = [
+        1, < float > SQRT2,
+        1, < float > SQRT2,
+        1, < float > SQRT2,
+        1, < float > SQRT2
```

Comment:
Just curious if this is recommended Python formatting being applied to cython? Does it make sense to format these differently given context?

Reply:
yeah, I was accidentally autopep8-ing cython files. Turned this off but I guess these snuck through.